swtbot_test/src/testsuites/ChangeRTOSVersion.java

```diff
@@ -37,18 +40,12 @@ public static void beforeClass() throws Exception {
 		bot = new SWTWorkbenchBot();
 		PlatformModel.loadPlatformModel(new File(Utility.getBundlePath(LogUtil.PLUGIN_ID, PLATFORM_XML_FILE)));
 		RTOSManager.loadRTOSModel(new File(Utility.getBundlePath(LogUtil.PLUGIN_ID, RTOS_PG_XML_FILE)));
-		Collection<ProjectModel> list = PGUtility.prepareProjectModel(RTOSType.AZURE, RTOSVersion.Azure_6_2_1,"changeRTOSVersion");
-		if(list.size()==1) {
-			for(ProjectModel model:list) {
-				projectModelSpecific=model;
-			}
-		}
+		projectModelSpecific = PGUtility.prepareProjectModel(RTOSType.AZURE, RTOSVersion.Azure_6_2_1, RTOSApplication.AZURE_USBX_CDC, Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_CK_RX65N);
 	}
 	
 	@Test
 	public void tc_01_CreateUsbxProject() throws Exception{
```

Comment:
E: the TC only checks for components after changing the version, it does not show that rtos version is actually changed.
Need to check version too.